libcxx/docs/Status/SpecialMath.rst

```diff
@@ -0,0 +1,35 @@
+.. special-math-status:
```

Comment:
As mentioned in our call this file needs to be added to https://github.com/llvm/llvm-project/blob/main/libcxx/docs/index.rst?plain=1

Reply:
Fixed in 84b7b87e968831265f000f008016fbfd3473f258.